utils/artifacts.py

```diff
@@ -53,10 +51,16 @@ def get_release_and_version(package_path: str) -> Tuple[bool, str, str, str, str
     """
     Load version and release info from deepsparse package
     """
+    # deepsparse/src/deepsparse/version.py always exists, default source of truth
+    version_path = os.path.join(package_path, "version.py")
```

Comment:
Also from looking at version.py, it seems like it also pulls version from generated_version.py if it exists - does that need to be changed as well?

Reply:
This is the intended behavior still, look at my previous response to lay out the flow a bit more.